src/main/java/seedu/address/model/entity/body/BodyIdentificationNumber.java

```diff
@@ -0,0 +1,69 @@
+package seedu.address.model.entity.body;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+import java.util.Objects;
+import java.util.regex.Pattern;
+
+//@@author ambervoong
+/**
+ *  Represents a body's identification number in Mortago.
+ */
+public class BodyIdentificationNumber {
+
+    public static final String VALID_ID = "A valid body identification number has a length of 9, "
+            + "starts with 'B', which is followed by 8 digits.";
+    private final String identificationNumber;
+
+    public BodyIdentificationNumber(String identificationNumber) {
```

Comment:
for identificationNumber, should we let the use just type in a number and then we append B and 0s in front? For example, the user just types "2" but it is stored as "B00000002"

Reply:
Sure! Great point, I'll add that in when I have the time.